bundles/AdminBundle/Resources/public/js/pimcore/helpers.js

```diff
@@ -3180,3 +3180,19 @@ pimcore.helpers.getProgressWindowListeners = function () {
         }
     };
 };
+
+pimcore.helpers.reloadUserImage = function (userId) {
+    var image = "/admin/user/get-image?id=" + userId + "&_dc=" + Ext.Date.now();
```

Comment:
Should also use JS Routing
```suggestion
    var image = Routing.generate('pimcore_admin_user_getimage', {id: userId, '_dc': Ext.Date.now()});
```

Reply:
Thanks